### PR TITLE
Added answers for the questions.

### PR DESCRIPTION
**Pull request by sohmpatil**
@sanket8397 Please review the branch and add comments.

### CODE REVIEW

File: answer.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*.jar
+*.war
+*.ear
+*.nar
+
+2. What does fast-forward in git mean? 
+Ans:Fast-forward merges literally move your main branch's tip forward to the end of your feature branch. This keeps all commits created in your feature branch sequential while integrating it neatly back into your main branch.
+
+3. Why should you have different branches? 
+Ans:For multiple people working in parallel, it is really necessary to track their work at different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less conphusion.
```
Comment (sanket8397) [inline answer.txt]: @sohmpatil  Check spelling of confusion.

File: answer.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+*.ear
+*.nar
+
+2. What does fast-forward in git mean? 
+Ans:Fast-forward merges literally move your main branch's tip forward to the end of your feature branch. This keeps all commits created in your feature branch sequential while integrating it neatly back into your main branch.
+
+3. Why should you have different branches? 
+Ans:For multiple people working in parallel, it is really necessary to track their work at different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less conphusion.  
+
+4. In a project how many remote Git repositories do you usually have?
```
Comment (sanket8397) [inline answer.txt:16]: @sohmpatil make it clear.

**Review by sanket8397 — CHANGES_REQUESTED** [review]
Please redo required changes.

File: answer.txt (hunk 1 above)
Comment (sohmpatil) [reply answer.txt]: @sanket8397 Updated with correct spelling.

File: answer.txt (hunk 2 above)
Comment (sohmpatil) [reply answer.txt:16]: @sanket8397 Updated the answer.

**Review by sanket8397 — APPROVED** [review]
Looks good to me.

**Pull request merged**